chart/apisix/values.yaml

```diff
@@ -190,5 +191,7 @@ nodeSelector: {}
 
 tolerations: []
 
+# It sets the anti-affinity of the Pod by default, if the affinity.enabled is set to true. 
+# If you want to set other affinity configuration, please modify the affinity item in tmplate/deployment.yaml
```

Comment:
This is very unfriendly to users for modify `deployment.yaml`.

`affinity.enabled` is not a good solution. We should add like `antiAffinity.enabled` to instead of it.

Reply:
good idea, I will rewrite like you speak.